src/analytics/pageResTimeEvent.ts

```diff
@@ -0,0 +1,42 @@
+import { AnalyticsEvent } from "./analyticsEvents";
+import { trackCustomEvent } from "./customEventTracker";
+
+export function trackPageResponseTime(): void {
+  window.addEventListener("load", () => {
```

Comment:
remove the event listener after use else keep adding listener?

so const t = () => {       
   windows.removeEventListern(...t );
}

windows.addEventListener("load", t);

May be I do not have enough knowledge to comment but this is single page app so there is only one load?